index.js

```diff
@@ -267,6 +267,40 @@ client.on("voiceStateUpdate", (oldState, newState) => {
             delete callRecords[userId];
         }
     }
+
+    // Check emojis for deleting todo:
+    const message = reaction.message.content;
+    const regex = /<@(\d+)>/g;
+
+    if (reaction.emoji.name === deleteTodoEmoji) {
+        if (message.match(regex)[0] === `<@${user.id}>`) {
+            deleteTodo[0] = true;
+        }
+        if (message.match(regex)[1] === `<@${user.id}>`) {
+            deleteTodo[1] = true;
+        }
+        if (deleteTodo[0] === true && deleteTodo[1] === true) {
```

Comment:
No need to compare booleans with `true`. Can simply do `deleteTodo[0] && deleteTodo[1]`